govcd/sample_govcd_test_config.yaml

```diff
@@ -70,10 +70,10 @@ vcd:
     disk:
       #
       # Disk size (bytes) for create disk, skip disk tests if it is less than or equal to 0
-      size: 1024
+      size: 1048576
       #
       # Disk size (bytes) for update disk, skip some disk tests if it is less than or equal to 0
-      sizeForUpdate: 10240
+      sizeForUpdate: 1048576
```

Comment:
How did the tests pass previously with disk size of less than an MB?

Reply:
Comment what is size 1048576 means would be good